context/src/main/java/io/micronaut/scheduling/ScheduledExecutorTaskScheduler.java

```diff
@@ -78,6 +80,39 @@ public ScheduledExecutorTaskScheduler(@Named(TaskExecutors.SCHEDULED) ExecutorSe
         return new ReschedulingTask<>(command, this, delaySupplier);
     }
 
+    @Override
+    public ScheduledFuture<?> schedule(String cron, String zoneId, Runnable command) {
```

Comment:
New methods are required. Per the comment in https://github.com/micronaut-projects/micronaut-core/pull/6939/files#r811273670 -

> This is a public method and thus cannot be changed. A new method will have to be created instead

Reply:
The old methods should just call this new one to avoid duplication

Reply:
Is this a reversal? In any event, happy to do it. Sorry on the delay.